tests/test_main.py

```diff
@@ -12,7 +12,8 @@
 
 import pretend
 
-from twine import __main__ as dunder_main, cli, exceptions
+from twine import __main__ as dunder_main
+from twine import cli, exceptions
```

Comment:
Is there anyway to force isort to put each import on it's own line? That will reduce merge conflict issues and reduce confusion for contributors

Reply:
Done in https://github.com/pypa/twine/pull/576/commits/5da1ea163c899f37db93d60b6f9ea63e21378aaf. The `typing` lines feel weird, but 🤷‍♂ . We can change it with the next release of isort if we want: https://github.com/timothycrosley/isort/commit/ad49f79cfd88807190e20d664a799e8a660a79d8